shell/shell_test.go

```diff
@@ -45,13 +45,13 @@ func TestMenderShellExecShell(t *testing.T) {
 	}
 
 	//command does not exist
-	pid, pseudoTTY, cmd, err := ExecuteShell(uint32(uid), uint32(gid), "thatissomethingthatdoesnotexecute", "xterm-256color", 24, 80)
+	pid, pseudoTTY, cmd, err := ExecuteShell(uint32(uid), uint32(gid), "/", "thatissomethingthatdoesnotexecute", "xterm-256color", 24, 80)
 	assert.Error(t, err)
 	assert.Equal(t, pid, -1)
 	assert.Nil(t, pseudoTTY)
 	assert.Nil(t, cmd)
 
-	pid, pseudoTTY, cmd, err = ExecuteShell(uint32(uid), uint32(gid), "/bin/sh", "xterm-256color", 24, 80)
+	pid, pseudoTTY, cmd, err = ExecuteShell(uint32(uid), uint32(gid), "/", "/bin/sh", "xterm-256color", 24, 80)
```

Comment:
could we have some explicit tests that it works? and that fallback works if the directory does not exist?
high-end test scenario: start terminal with given home, run the pwd command, compare the results. (copy paste from one of the tests that checks command execution)

Reply:
Added a unit test